Zend/zend_compile.c

```diff
@@ -1255,7 +1255,7 @@ static zend_bool is_generator_compatible_class_type(zend_string *name) {
 		|| zend_string_equals_literal_ci(name, "Generator");
 }
 
-static void zend_mark_function_as_generator() /* {{{ */
+static void zend_mark_function_as_generator(void) /* {{{ */
```

Comment:
Did these actually warn? These are not definitions, which have different semantics from declarations.

Reply:
Yeah at least on Travis, but maybe it's due to something else

Reply:
So I needed to look up what the exact difference is between declaration and definition is in C.
From what I understand if there is no prototype to act as a declaration then the definition also acts as the declaration of the function, that's why it warns *I think*